packages/sdk-middleware-auth/src/base-auth-flow.js

```diff
@@ -0,0 +1,133 @@
+/* @flow */
+import type {
+  MiddlewareRequest,
+  Next,
+  Task,
+  AuthMiddlewareBaseOptions,
+} from 'types/sdk'
+
+/* global fetch */
+import 'isomorphic-fetch'
+
+
+export default function authMiddlewareBase ({
+    request,
+    response,
+    url,
+    basicAuth,
+    body,
+    pendingTasks,
+    requestState,
+    tokenCache,
+  }: AuthMiddlewareBaseOptions,
+  next: Next,
+) {
+  // Check if there is already a `Authorization` header in the request.
+  // If so, then go directly to the next middleware.
+  if (
+    (request.headers && request.headers['authorization']) ||
+    (request.headers && request.headers['Authorization'])
```

Comment:
Because it's can be an issue that occurred from the http module used when fetching OAuth token from the API